cs2pr

```diff
@@ -31,6 +32,9 @@ foreach ($argv as $arg) {
         case 'graceful-warnings':
             $gracefulWarnings = true;
             break;
+        case 'show-source':
```

Comment:
Thx for the PR.

I feel this option should be named `prepend-source` or similar.
Additionally the new option should be added to the help output.